<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolForm</class>
 <widget class="QWidget" name="ToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>63</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QPushButton" name="fullScreenBtn">
     <property name="toolTip">
      <string>full screen</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="expandNotifyBtn">
     <property name="toolTip">
      <string>expand notify</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeScreenBtn">
     <property name="toolTip">
      <string>close screen</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="powerBtn">
     <property name="toolTip">
      <string>power</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="volumeUpBtn">
     <property name="toolTip">
      <string>volume up</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="volumeDownBtn">
     <property name="toolTip">
      <string>volume down</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="appSwitchBtn">
     <property name="toolTip">
      <string>app switch</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="menuBtn">
     <property name="toolTip">
      <string>menu</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="homeBtn">
     <property name="toolTip">
      <string>home</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="returnBtn">
     <property name="toolTip">
      <string>return</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
